tests/unit/conf/Config.js

```diff
@@ -1,13 +1,14 @@
 'use strict'; // eslint-disable-line
 
-const assert = require('assert');
+process.env.SITE_NAME = 'aws-location';
```

Comment:
Doing this at the top-level will always filter the bootstrap list, don't we want to test both cases?

If always testing with filtering is fine, then we would prefer having this env variable in the github action step, so we do not need to alter the config at runtime (depending on the order of imports, it may also not always work).